packages/ui/ToggleItem/ToggleItem.js

```diff
@@ -78,11 +79,11 @@ const ToggleItemBase = kind({
 		 * and must therefore respond to it in some way. It is recommended to use
 		 * [ToggleIcon]{@link ui/ToggleIcon} for this.
 		 *
-		 * @type {Component|Element}
+		 * @type {String|Component|Element}
 		 * @required
 		 * @public
 		 */
-		iconComponent: PropTypes.oneOfType([PropTypes.string, PropTypes.element, PropTypes.func]).isRequired,
+		iconComponent: EnactPropTypes.renderableOverride.isRequired,
```

Comment:
This one really shouldn't be a plain string as it receives props.  What was the reasoning for adding `String` here?  Also, other `iconComponents` are generally just `Component`, but this one is also `Element`.